File: scheduling/scheduling-annotation-processor/src/test/java/ru/tinkoff/kora/scheduling/annotation/processor/controller/ScheduledWithTrigger.java
package ru.tinkoff.kora.scheduling.annotation.processor.controller;

import org.quartz.JobExecutionContext;
import ru.tinkoff.kora.common.Tag;
import ru.tinkoff.kora.scheduling.quartz.ScheduleWithTrigger;

public class ScheduledWithTrigger {
    @ScheduleWithTrigger(@Tag(ScheduledWithTrigger.class))
    public void noArgs() {}

    @ScheduleWithTrigger(@Tag(ScheduledWithTrigger.class))
    public void withCtx(JobExecutionContext jobExecutionContext) {}
}

File: scheduling/scheduling-ksp/src/test/kotlin/ru/tinkoff/kora/scheduling/ksp/controller/ScheduledWithTrigger.kt
package ru.tinkoff.kora.scheduling.ksp.controller

import org.quartz.JobExecutionContext
import ru.tinkoff.kora.common.Tag
import ru.tinkoff.kora.scheduling.quartz.ScheduleWithTrigger

class ScheduledWithTrigger {
    @ScheduleWithTrigger(Tag(ScheduledWithTrigger::class))
    fun noArgs() {
    }

    @ScheduleWithTrigger(Tag(ScheduledWithTrigger::class))
    fun withCtx(jobExecutionContext: JobExecutionContext?) {
    }
}
